tasks/main.yml

```diff
@@ -162,6 +162,54 @@
       tags: configure
   when: openio_redis_configuration_split
 
+- block:
+    - name: Ensure backup name is valid
+      assert:
+        that: 'item.name | d("") | regex_search("^[a-zA-Z0-9_\.-]+$")'
+        fail_msg: 'Backup name is not valid (only letters, numbers, dash, underscore and dot) for "{{ item.name }}"'
+      with_items: "{{ openio_redis_backups }}"
+
+    - name: Ensure backup cron is valid
+      assert:
+        that: 'item.cron | d("") | regex_search("^\s*(@(annualy|yearly|monthly|weekly|daily|hourly|reboot)|((\d+(-\d+)?|\*)(/\d+)?(,(\d+(-\d+)?|\*)(/\d+)?)*\s+){3}((\d+|jan|feb|mar|apr|may|jun|jul|aug|sep|oct|nov|dec)(-(\d+|jan|feb|mar|apr|may|jun|jul|aug|sep|oct|nov|dec))?|\*)(/\d+)?(,((\d+|jan|feb|mar|apr|may|jun|jul|aug|sep|oct|nov|dec)(-(\d+|jan|feb|mar|apr|may|jun|jul|aug|sep|oct|nov|dec))?|\*)(/\d+)?)*\s+((\d+|mon|tue|wed|thu|fri|sat|sun)(-(\d+|mon|tue|wed|thu|fri|sat|sun))?|\*)(/\d+)?(,((\d+|mon|tue|wed|thu|fri|sat|sun)(-(\d+|mon|tue|wed|thu|fri|sat|sun))?|\*)(/\d+)?)*)\s*$")'
```

Comment:
max 120 chars
Please respect this rule:
```
foo: " bar \
          | filter1 \
          | default('plop') "
```

Reply:
or make your own python filter

Reply:
fixed